### DISCUSSION TITLE
a bug? in the calculations window.

### DISCUSSION THREAD

**Discussion by tzmosh** [Stellarium/stellarium#3019] [General]
In the Astronomical calculations window, Under the Phenomena tab, the perihelion and aphelion option appears. I don't see that it appears in the results of the calculations.

**Comment by ilkant** [comment]
I tried several planets and there are no oppositions, elongations and quadratures after I pushed calculate phenomena.

**Comment by tzmosh** [comment]
In my understanding, phenomena between objects whose angular distance between them is greater than 20 degrees of arc are not calculated.
Therefore, in a certain date range, phenomena will not always be found.
As for the problem I reported, these are phenomena that are included in the range, and should have been calculated.
